dags/openshift_nightlies/config/benchmarks/large-control-plane-mgs.json

```diff
@@ -0,0 +1,40 @@
+{
+    "benchmarks": [
+        {
+            "name": "node-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=ripsaw-kube-burner"
+        },
+        {
+            "name": "cluster-density",
+            "workload": "kube-burner",
+            "trigger_rule": "all_done",
+            "command": "./run.sh",
+            "env": {
```

Comment:
Is this still needed with the wrapper? I think kube-burner will set most of these (not read from them)

https://github.com/cloud-bulldozer/kube-burner/blob/master/pkg/workloads/cluster-density.go#L43-L48

Reply:
ack